src/Microsoft.PowerApps.TestEngine/System/FileSystem.cs

```diff
@@ -499,5 +499,20 @@ public bool IsWritePermittedFilePath(string filePath)
                 return false;
             }
         }
+
+        public void Delete(string fileName)
+        {
+            if (!FileExists(fileName))

```

Comment:
can we add additional check for IsWritePermittedFilePath, this will ensure file can be deleted only from place where write is allowed

Reply:
Updated to include IsWritePermittedFilePath